doc/news/changes/minor/20180715DavidWells

```diff
@@ -0,0 +1,4 @@
+New: A new function GridGenerator::concentric_hyper_shells has been added. This
+function generates a grid comprised of shells of varying widths, where the
+shells near the center are much thinner than those near the outer boundary.
+<br> (David Wells, 2018/07/15)
```

Comment:
I guess this needs a line break between `<br>` and your name. Sorry I didn't pick this up earlier.

Reply:
Fixed!